python_modules/libraries/dagster-dbt/dagster_dbt/asset_decorator.py

```diff
@@ -53,6 +55,8 @@ def dbt_assets(
             to include. Defaults to ``fqn:*``.
         exclude (Optional[str]): A dbt selection string for the models in a project that you want
             to exclude. Defaults to "".
+        selector (Optional[str]): A dbt selector for the models in a project that you want to
```

Comment:
should we be throwing an explicit error here if users use some combination? Or does dbt handle that?

Reply:
We should throw an error - it is not stated clearly in the [dbt selector guide](https://docs.getdbt.com/reference/node-selection/yaml-selectors), but the selection passed via `--selector` overrides anything passed via `--select` or `--exclude` in the same command.